src/compounds/ad-banner/src/index.tsx

```diff
@@ -22,15 +22,17 @@ interface Props extends React.HTMLAttributes<HTMLDivElement> {
   href: string
   imageCritical?: boolean
   className?: string
+  targetAttribute?: string
+  uspColor?: string
 }
 
 const AdBanner: React.FC<Props> = ({
   title,
   content,
   usp,
   sponsor,
-  backgroundColor = 'black',
-  textColor = 'white',
+  backgroundColor = '',
+  textColor = '',
```

Comment:
if you're doing `textColor || white` in a few places, shouldn't this just be `textColor = 'white'` to simplify that?

Reply:
I had to do it this way because in our frontend we may be passing textColor = null to the component (as text colour is optional in contentful) and in this case that would override the default textColor = 'white'